README.md

```diff
@@ -43,6 +43,7 @@ Awesome software projects sub-categorized by focus.
 - [bh_tomo](https://github.com/groupeLIAMG/bh_tomo) – ![matlab](media/icon/matlab.svg) Borehole Radar and Seismic Tomography Package
 - [gprMax](http://www.gprmax.com/) – ![Python](media/icon/python.svg) ![CUDA](media/icon/cuda.svg) Finite-difference time-domain electromagnetic wave propagation simulator. gprMax can be run on either CPU or GPU. 
 - [modelr.io](https://www.modelr.io/) – ![Python](media/icon/python.svg) ![Javascript](media/icon/javascript.svg) Web app for simple synthetic seismic forward modeling.
+- [ModFlow](https://water.usgs.gov/ogw/modflow/MODFLOW.html) - Flow modelling software distributed by the USGS to simulate and predict groundwater conditions and groundwater/surface-water interactions, also comes with additional [variants and add-ons] (https://water.usgs.gov/ogw/modflow/).  
```

Comment:
Hi @didiooi thanks for your contributions! Could you please remove the space between `]` and `(` in the last link? Github won't render the link properly because of it. Thanks!

Reply:
@didiooi and @leouieda, Geospatial stuff is a whole beast unto itself. I agree that some good power tools should be included here. But at some point we might just want to link to an awesome list for geospatial, like this one. 

https://github.com/sacridini/Awesome-Geospatial

Thoughts?

Reply:
Well, I feel kind of bad rejecting something when @didiooi put in the work to contribute. We should probably make it clear that geospacial contribs should go to Awesome-Geospacial but I'd be OK with accepting this PR. Particularly because ModFlow is not really Geospacial (I don't get this division, actually).

Reply:
Not a problem! I believe I had Modflow under Simulation, it's definitely not for geospatial - let me check. No problem on rejecting anything do what is best for Software Underground!